fuse_core/include/fuse_core/transaction.h

```diff
@@ -208,6 +208,13 @@ class Transaction
    */
   void merge(const Transaction& other, bool overwrite = false);
 
+  /**
+   * @brief Check if the transaction is empty, i.e. it doesn't have any variable or constraint added or removed.
+   *
+   * @return True if the transaction is empty.
+   */
+  bool empty() const;
```

Comment:
Initially I thought of doing `.stamp().isZero()` instead of `empty()`, but the `transaction` I get in the `notifyCallback` at the publisher is the merged transaction, which doesn't have the `stamp_` set, so it's always **zero**. See https://github.com/locusrobotics/fuse/pull/79